store/src/main/java/org/apache/rocketmq/store/MappedFile.java

```diff
@@ -570,6 +570,11 @@ public void munlock() {
         log.info("munlock {} {} {} ret = {} time consuming = {}", address, this.fileName, this.fileSize, ret, System.currentTimeMillis() - beginTime);
     }
 
+    //testable
+    File getFile() {
+        return this.file;
```

Comment:
LGTM, but no need to return this file to test code, you could structure the File object directly.